packages/ui-signer/src/types.d.ts

```diff
@@ -9,52 +9,52 @@ import { Base, UInt } from '@polkadot/types/codec';
 import { Hash, Extrinsic } from '@polkadot/types';
 import { RawParam$Value } from '@polkadot/ui-app/Params/types';
 
-export type QueueTx$Status = 'finalised' | 'usurped' | 'dropped' | 'broadcast' | 'cancelled' | 'completed' | 'error' | 'incomplete' | 'queued' | 'sending' | 'sent';
+export type AccountInfo = {
+  accountId?: string | null,
+  accountNonce: UInt | BN
+};
+
+export type QueueTx$Status = 'finalised' | 'usurped' | 'dropped' | 'broadcast' | 'cancelled' | 'completed' | 'error' | 'incomplete' | 'queued' | 'sending' | 'sent' | 'blocked';
 
 export type QueueTx$Id = number;
 
-export type QueueTx$Result = {
+export type QueueTx = AccountInfo & {
   error?: Error,
+  extrinsic?: Extrinsic,
+  id: QueueTx$Id,
   result?: any,
+  rpc: RpcMethod,
+  values?: Array<any>,
   status: QueueTx$Status
 };
 
-export type AccountInfo = {
-  accountId?: string | null,
-  accountNonce: UInt | BN
-};
-
-export type ParialAccountInfo = {
-  accountId?: string | null,
-  accountNonce?: UInt | BN | null
+export type QueueTx$Result = {
+  error?: Error,
+  result?: any,
+  status: QueueTx$Status
 };
 
 export type QueueTx$Extrinsic = AccountInfo & {
   extrinsic: Extrinsic
-}
-
-export type PartialQueueTx$Extrinsic = ParialAccountInfo & {
-  extrinsic: Extrinsic
 };
 
 export type QueueTx$Rpc = AccountInfo & {
   rpc: RpcMethod,
   values: Array<any>
 };
 
-export type PartialQueueTx$Rpc = ParialAccountInfo & {
-  rpc: RpcMethod,
-  values: Array<any>
+export type PartialAccountInfo = {
+  accountId?: string | null,
+  accountNonce?: UInt | BN | null
```

Comment:
As above, just `BN | null` (since we are making changes, not that anybody would have guessed this)